### ISSUE TITLE
problems with pmag_gui Windows install

### ISSUE THREAD

**Issue by jbowles100** [PmagPy/PmagPy#641]
We're trying to install the pmag_gui on the Geosciences computer lab Windows machines running Windows 10 Educational (v. 1909).  The executable launches, but then stalls out with two message windows open: "Reading in data from current working directory, please wait..." (note that directory is empty); and "Failed to execute script 'pmag_gui'".  Text in the stdout/stderr window is attached.  The magic_gui appears to be OK, although both guis fail to launch from the command line.
[PmagPy errors.docx](https://github.com/PmagPy/PmagPy/files/7864970/PmagPy.errors.docx)

**Comment by jbowles100** [comment]
Thanks.  I’ll check back with IT and see this can resolve it.

Julie

**Comment by Swanson-Hysell** [comment]
Hey @jbowles100. Were you able to come up with a solution here?

**Comment by paselkin** [comment]
I'm having a similar issue in Windows 11 (Windows 11 Education Version 10.0.22000 Build 22000) - pmag_gui launches, but stalls out while "Reading in data from current working directory, please wait..." I don't see another window with an error message. 

Using a fresh pip install of PmagPy from today - not sure what version of pmag_gui is in it.

**Comment by Swanson-Hysell** [comment]
Interesting. We can test this on a Windows machine here and see if we can reproduce the issue.

Is using the standalone executable version of the program a possible solution?  https://github.com/PmagPy/PmagPy-Standalone-Windows/releases/tag/4.2.72b

**Comment by paselkin** [comment]
The standalone stalls out while "Compiling required data, please wait..." with the following errors in the log:
![image](https://user-images.githubusercontent.com/4691430/217111310-54e353e7-5ad8-42d7-852f-c7f31ca7452b.png)

**Comment by duserzym** [comment]
I was able to reproduce this issue on our windows 10 lab computer. 

I recompiled the Windows standalone with the latest pmagpy release. For our lab computer the issue is solved. I tested demag_gui and Thellier_gui and they seem to work now. The executable file can be downloaded here:https://github.com/PmagPy/PmagPy-Standalone-Windows/releases/tag/4.2.109

@paselkin @jbowles100 could you please try this new version and let me know if this is working?

**Comment by paselkin** [comment]
Yay! The new executable works. Any chance we can figure out why the version with the CLI tools in the conda package doesn't work? Was it an issue with the version of wxpython? Or is there a new version where it works now?

**Comment by paselkin** [comment]
Maybe I spoke too soon. I'm trying to change the paleointensity quality parameters included in the output and get this message instead of the pop-up window:

> Traceback (most recent call last):
>   File "programs\thellier_gui.py", line 2286, in on_menu_m_preferences_spd
>   File "dialogs\thellier_gui_dialogs.py", line 143, in __init__
>   File "dialogs\thellier_gui_dialogs.py", line 209, in InitUI
> wx._core.wxAssertionError: C++ assertion "CheckSizerFlags(!((flags) & (wxALIGN_CENTRE_HORIZONTAL)))" failed at ..\..\src\common\sizer.cpp(2279) in wxBoxSizer::DoInsert(): wxALIGN_CENTRE_HORIZONTAL will be ignored in this sizer: only vertical alignment flags can be used in horizontal sizers
> 
> DO NOT PANIC !!
> 
> If you're an end user running a program not developed by you, please ignore this message, it is harmless, and please try reporting the problem to the program developers.
> 
> You may also set WXSUPPRESS_SIZER_FLAGS_CHECK environment variable to suppress all such checks when running this program.
> 
> If you're the developer, simply remove this flag from your code to avoid getting this message. You can also call wxSizerFlags::DisableConsistencyChecks() to globally disable all such checks, but this is strongly not recommended.

**Comment by duserzym** [comment]
Which function/dialog window exactly are you trying to open? Changing the acceptance criteria? Could you provide a screenshot, and perhaps provide some example data and steps to help me reproduce this issue?

It looks like a wxpython update related issue and there could be a quick fix.

**Comment by paselkin** [comment]
I tried to select paleointensity statistics to view/save using Preferences > Specimen Paleointensity Statistics (From SPD List) when I got the above error. In the working copy on my old machine, if I remember right, this would cause a window to pop up with a list of paleointensity statistics.

**Comment by duserzym** [comment]
Thanks for the instructions. I changed ``wxALIGN_CENTRE_HORIZONTAL `` to ``wxALIGN_CENTRE_VERTICAL `` in the code and it seems to work now...! I just updated the release page: https://github.com/PmagPy/PmagPy-Standalone-Windows/releases/tag/4.2.109. Let me know if this is working! 

Given that this was not an issue for the mac compilation, I am hesitating on committing this change to the main PmagPy repo now. I will seek to discuss this issue with the rest of the PmagPy contributors.
